### PR TITLE
Scheduler Editing demo rewrite

### PR DESCRIPTION
**Pull request by pavel-2020**
[Trello](https://trello.com/c/5OCaftyh/1666-scheduler-demos-editing-1-demo)
[Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/Editing/jQuery/Light/)
CPs: https://github.com/DevExpress/devextreme-demos/pull/933, https://github.com/DevExpress/devextreme-demos/pull/934

### CODE REVIEW

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/Scheduler/Editing/description.md [hunk 1]
```diff
@@ -1 +1,13 @@
-This example demonstrates appointment editing. You can add, update and delete appointments, as&nbsp;well as&nbsp;modify them by&nbsp;dragging or&nbsp;resizing. Toggle the check boxes under the Scheduler to&nbsp;define which editing features are enabled.
\ No newline at end of file
+Scheduler allows users to add, update, resize, drag, and delete appointments. To control these operations, specify properties in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this demo, you can use the checkboxes below the Scheduler to toggle the edit operations.
+
+These operations raise events that you can handle with the following functions:
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Scheduler/Editing/description.md:3]: I can see only event names in the following list. I can see no functions.

File: JSDemos/Demos/Scheduler/Editing/description.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+* [onAppointmentAdding](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentAdding) / [onAppointmentAdded](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentAdded)
+
+* [onAppointmentDeleting](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentDeleting) / [onAppointmentDeleted](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentDeleted)
+
+* [onAppointmentUpdating](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentUpdating) / [onAppointmentUpdated](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentUpdated)
+
+In this demo, we have configured the handlers to display a toast message after a user performs an edit operation.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Scheduler/Editing/description.md]: Stick to past simple.

```suggestion
In this demo, we configured the handlers to display a toast message after a user performs an edit operation.
```

File: JSDemos/Demos/Scheduler/Editing/description.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+
+Users can edit appointment data in the appointment details form. To open it, they need to double-click the appointment. You can use the [onAppointmentFormOpening](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentFormOpening) function to customize the form.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Scheduler/Editing/description.md:13]: What is `onAppointmentFormOpening`? Is it a function? Looks like an event's name.

File: JSDemos/Demos/Scheduler/Editing/description.md (hunk 1 above)
Comment (pavel-2020) [reply JSDemos/Demos/Scheduler/Editing/description.md:3]: Functions that start with "on" are event handlers, e.g. onAppointmentAdding is a function, appointmentAdding is an event. [StackOverflow explanation](https://softwareengineering.stackexchange.com/questions/299101/what-does-the-prefix-on-in-a-function-e-g-onbuttonpressed-mean).

File: JSDemos/Demos/Scheduler/Editing/description.md (hunk 3 above)
Comment (pavel-2020) [reply JSDemos/Demos/Scheduler/Editing/description.md:13]: My prev. message: https://github.com/DevExpress/devextreme-demos/pull/858/files#r639458623

File: JSDemos/Demos/Scheduler/Editing/description.md [hunk 4]
```diff
@@ -1 +1,13 @@
-This example demonstrates appointment editing. You can add, update and delete appointments, as&nbsp;well as&nbsp;modify them by&nbsp;dragging or&nbsp;resizing. Toggle the check boxes under the Scheduler to&nbsp;define which editing features are enabled.
\ No newline at end of file
+Scheduler allows users to add, update, resize, drag, and delete appointments. To control these operations, specify properties in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this demo, you can use the checkboxes below the Scheduler to toggle the edit operations.
```
Comment (albertov05) [inline JSDemos/Demos/Scheduler/Editing/description.md:1]: 1) `To control these operations, specify properties in the`...

What do you mean by "control these operations"? What are you controlling?

2) `In this demo, you can use the checkboxes below the Scheduler to toggle the edit operations.`

When you say "toggle the edit operations", are you specifying which operations (add, update, resize, drag, delete) are available to the user? For example, do you mean "...to toggle the edit operations available to users."

Comment (pavel-2020) [reply JSDemos/Demos/Scheduler/Editing/description.md:1]: 1) Controlling whether they are disabled or enabled (enabled by default).
2)
A user can specify which operations to enable:
image

Comment (albertov05) [reply JSDemos/Demos/Scheduler/Editing/description.md:1]: Thank you for the clarification.

**Pull request merged**